### ISSUE TITLE
VocabSieve : AppImage

### ISSUE THREAD

**Issue by Exponent4806** [Blackdeer1524/Dict2Flashcards#2]
Hi ! It would be useful if you check the project "VocabSieve".  The projects are very similar.

https://github.com/FreeLanguageTools/vocabsieve/

I want to try your idea but on Linux seems much easier with VocabSieve. Perhaps could you also get new ideas from that project and implement there here ? 

An AppImage would be useful on Linux... ! 

Thanks for your good work...! You can also help in VocabSieve...!

**Comment by ApprobationBW** [comment]
Thanks for your kind reply and effort !

**Comment by ApprobationBW** [comment]
On the other hand, it seems that we can use folders with local audios in your software. That is a really good idea. All the audios from Forvo are available at FreeMDict Forum: https://forum.freemdict.com/t/topic/12050 

Another idea would be to integrate some dictionaries into the software (.dsl or .mdx format).

Thanks for your great effort ! Please keep up your good work :D

**Comment by Blackdeer1524** [comment]
> Thanks for your kind reply and effort ! My suggestion would be to make the software easier to "run" to people without a background in computer science. I tried to run it but failed and I have been using Manjaro Linux for over 3 years. The average "Windows" user would probably also fail to use the software.

I must agree that this project isn't very portable... At least for linux distros. I only tested it on ubuntu and windows 10/11, and I am not really aware of installation of nessesary packages on other linux distros. Definitely need to look into it. 

By the way, nessesary package are
- PyGObject
- TKinter

After you install them using your package manager everything should be working fine. There are also additional instructions in the `README` to help you set up this app with Anki. If you want to set up this project into virtual environment, then there is a file `installation/linux/installation_guide.txt` which describes the steps needed. I don't expect that this part isn't portable. 

> On the other hand, it seems that we can use folders with local audios in your software. That is a really good idea. All the audios from Forvo are available at FreeMDict Forum: https://forum.freemdict.com/t/topic/12050
> 
> Another idea would be to integrate some dictionaries into the software (.dsl or .mdx format).
> 
> Thanks for your great effort ! Please keep up your good work :D

There is actually support for local dictionaries / local audios in form of plugins. But yeah, one would need to **write** their own plugin for your dictionary / local audio. This approach isn't very user-friendly, so automated import of established formats would be a good addition.
